### ISSUE TITLE
For menus, when you get to the nested menu, the screen reader will leave the menu and start reading the underlying page. 

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#471]
# Bug Report

## Description
When using VoiceOver navigation with the left/right arrow keys, the screen reader will leave the menu and start reading the underlying page. 

## Steps to Reproduce

1. Go to: https://engineering.cerner.com/terra-ui/#/components/terra-menu/menu/menu
2. Turn VoiceOver on (Command F5)
3. VoiceOver starts reading the page content
4. Navigate the page using the left/right arrows and/or the Tab button
5. With focus on the Click Me button, hit Space or Enter to open the default menu example
6. VoiceOver reads the menu content
7. Use the Tab and Shift+Tab keys to move through the menu (should work fine and focus stays within the menu and submenus, as expected)
8. Use the left and right arrow (VO navigation) keys instead (focus leaves the menu and navigation continues on the underlying page, even though the menu is still open)

## Additional Context / Screenshots
Identified as part of functional verification on [this PR](https://github.com/cerner/terra-core/pull/2004#issuecomment-438396868).

## Expected Behavior
The screen reader will not leave the menu and start reading the underlying page unless the menu is closed.

## Environment

* Component Name and Version: terra-menu

**Comment by ryanthemanuel** [comment]
Spoked to Mike offline about this.  It happens for all menu examples (nested or not).

**Comment by supreethmr** [comment]
This issue is due to additional react-fragment that has been wrapped around MenuContent in [_MenuContent.jsx](https://github.com/cerner/terra-framework/blob/9bb040d699bda468074c61c5f947f73867f9647d/packages/terra-menu/src/_MenuContent.jsx#L345) . 

With my understanding of react-fragment I feel wrapping the react-fragment around menu-content will make menu items (ul and li) non-accessible for VO. Since it wraps multiple element together VO may not be able to recognize menu-items as list.

Reasons to point on React-Fragment of menu-content is :
The Default VO functionality of left and right arrow keys is to read the content character by character when pressed, But in this scenario left and right arrow keys are navigating through menu-items and Up arrow and Down arrow keys are not navigating through the menu-items.

So this issue might be due to  VO unable to recognize menu-items as list (li) and treating menu-dialog as div. 

Possible fix : 
Remove React-Fragment from _MenuContent.jsx : [code sample](https://github.com/cerner/terra-framework/commit/10acf9b443f35226a34ce43ab252bf685be53558)

Here is the screenshot of VO reading the menu contents on left and right key press After fix and also navigates to sub menu on right arrow key press which was not happening before fix : 

Right arrow key press on menu-item :
1. <img width="1680" alt="Screen Shot 2019-06-04 at 5 24 15 PM" src="https://user-images.githubusercontent.com/12869809/58877280-f0647f80-86ed-11e9-9d07-962ec86abefb.png">

2. <img width="1680" alt="Screen Shot 2019-06-04 at 5 24 19 PM" src="https://user-images.githubusercontent.com/12869809/58877308-fc504180-86ed-11e9-9c9a-c651b9c33e3b.png">

Right Arrow key press on menu-item having sub-menu : 
1. <img width="1680" alt="Screen Shot 2019-06-04 at 5 24 25 PM" src="https://user-images.githubusercontent.com/12869809/58877400-34f01b00-86ee-11e9-9646-60e2b0de9a97.png">

Left Arrow key press to navigate back to main menu : 
1. <img width="1680" alt="Screen Shot 2019-06-04 at 5 24 28 PM" src="https://user-images.githubusercontent.com/12869809/58877956-c57b2b00-86ef-11e9-9f3f-0890ced13961.png">
